### PR TITLE
Feature/use current branch an pullrequest on cli

### PR DESCRIPTION
**Pull request by mms-gianni**

### CODE REVIEW

File: gitcd/Git/Feature.py [hunk 1]
```diff
@@ -4,38 +4,69 @@
...
+    branch = self.getFeaturebranch(branch)
+
+    master = self.config.getMaster()
+
+    repo = self.cli.executeRaw("git remote show origin -n | grep h.URL | sed 's/.*://;s/.git$//'")
+    
+    token = self.config.getToken()
+
+    if token != None:
+      title = self.interface.askFor("Pullrequtest Title?")
```
Comment (claudio-walser) [inline gitcd/Git/Feature.py]: Pullrequtest? Geile Scheiss :D

File: gitcd/Git/Feature.py [hunk 2]
```diff
@@ -4,38 +4,69 @@
...
 
     origin = self.getOrigin()
     developmentBranch = self.getDevelopmentBranch()
+    branch = self.getFeaturebranch(branch)
+      
+
+    if branch == None:
+      branch = self.cli.executeRaw("git rev-parse --abbrev-ref HEAD")
+    else: 
+      branch = self.config.getFeature().branch
```
Comment (claudio-walser) [inline gitcd/Git/Feature.py]: De isch au kaputt :p
walsercl@tpcc-le-m00202: ~/Development/claudio/knack $ mpcd feature test -b test-feature
Which develop branch you want to use?
Possibilities: [develop, develop-2016-03-08]
Default: develop
develop-2016-03-08
Traceback (most recent call last):
  File "/usr/local/bin/mpcd", line 45, in <module>
    main(arguments.command, arguments.action, arguments.branch)
  File "/usr/local/bin/mpcd", line 38, in main
    gitcd.dispatch(command, action, branch)
  File "/Users/walsercl/Development/claudio/gitcd/gitcd/Gitcd.py", line 72, in dispatch
    method(branch)
  File "/Users/walsercl/Development/claudio/gitcd/gitcd/Git/Feature.py", line 33, in test
    branch = self.config.getFeature().branch
AttributeError: 'str' object has no attribute 'branch'

File: gitcd/Git/Feature.py [hunk 3]
```diff
@@ -4,38 +4,69 @@
...
 
     # todo: need to handle this as prefix and read all possible branches
     # ask user if more than one possibillities
     self.cli.execute("git checkout %s" % (developmentBranch))
     self.cli.execute("git pull %s %s" % (origin, developmentBranch))
-    self.cli.execute("git merge %s%s" % (self.config.getFeature(), branch))
+    self.cli.execute("git merge %s" % (branch))
     self.cli.execute("git push %s %s" % (origin, developmentBranch))
 
+    self.interface.ok("gitcd feature test: ".branch)
```
Comment (claudio-walser) [inline gitcd/Git/Feature.py]: Hesch PHP gschriebe? :D

walsercl@tpcc-le-m00202: ~/Development/claudio/knack $ mpcd feature test -b test-feature
Which develop branch you want to use?
Possibilities: [develop, develop-2016-03-08]
Default: develop
develop-2016-03-08
Traceback (most recent call last):
  File "/usr/local/bin/mpcd", line 45, in <module>
    main(arguments.command, arguments.action, arguments.branch)
  File "/usr/local/bin/mpcd", line 38, in main
    gitcd.dispatch(command, action, branch)
  File "/Users/walsercl/Development/claudio/gitcd/gitcd/Gitcd.py", line 72, in dispatch
    method(branch)
  File "/Users/walsercl/Development/claudio/gitcd/gitcd/Git/Feature.py", line 43, in test
    self.interface.ok("gitcd feature test: ".branch)
AttributeError: 'str' object has no attribute 'branch'

**Comment by mms-gianni** [comment]
yep... han de Fix nonig commited. sett es + sii anstatt emne .

**Pull request merged**